src/git.js

```diff
@@ -884,6 +884,24 @@
    };
 
    /**
+    *
+    * @param {Array} options
+    * @param {Function} [then]
+    */
+   Git.prototype.clean = function(options, then) {
+      if(!Array.isArray(options) || options.length == 0) {
```

Comment:
Please can this make use of the `Git.trailingFunctionArgument` utility to pick out the next handler function from the supplied arguments. As the `git clean` command can be called without arguments it would be useful to have the `options` argument be optional.